src/main/java/com/hillel/items_exchange/controller/ImageController.java

```diff
@@ -0,0 +1,39 @@
+package com.hillel.items_exchange.controller;
+
+import com.hillel.items_exchange.dto.ImageDto;
+import com.hillel.items_exchange.service.ImageService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class ImageController {
+    private final ImageService imageService;
+
+    @GetMapping("/adv/{advertisement_id}/product/{product_id}/image-urls")
```

Comment:
You get here images not advertisements so it should be '/image' global path here. I don't know why do you need advertisement here? Just use product and that's all and it would be like "/links/{product_id}".